.github/workflows/build.yml

```diff
@@ -19,19 +17,50 @@ jobs:
         with:
           java-version: ${{ matrix.java }}
       - name: Build with Maven
-        run: mvn -B package --file pom.xml
+        run: mvn -B package --no-transfer-progress --file pom.xml
```

Comment:
I noticed we don't run tests for neither workflows. Should we?

Reply:
What do you mean? package include test-phase:
`Tests run: 101, Failures: 0, Errors: 0, Skipped: 0`

Reply:
I could probably skip test in the release-build. But these tests are so fast, so I don't bother.

Reply:
Haha, ofc. Sorry about that. I blame it on not drinking any coffee yet.